File: TenMinsPy/day76.py
# create function addToCsv() that will create a csv file or append a row to an existing csv file, returning the row#

import os
import csv

class day76:
    def addToCsv():
        row = 1
        if os.path.exists("D:/coding30minsAday/textFiles/cvsDay76P.csv"):
            with open("D:/coding30minsAday/textFiles/cvsDay76P.csv", "a", newline="", encoding="utf-8") as file:
                w = csv.writer(file)
                w.writerow("")
                
            with open("D:/coding30minsAday/textFiles/cvsDay76P.csv", "r", newline="", encoding="utf-8") as file:
                row = sum(1 for _ in file) + 1
        else:
            with open("D:/coding30minsAday/textFiles/cvsDay76P.csv", "w", newline="", encoding="utf-8") as file:
                writer = csv.writer(file)
                writer.writerows("")
        return row

print(day76.addToCsv())

File: TenMinsJava/day76.java
// create function addToCsv() that will create a csv file or append a row to an existing csv file, returning the row#
package TenMinsJava;

import java.io.File;
import java.io.FileWriter;
import java.util.Scanner;

public class day76 {

    public static int addToCvs() {
        int row = 1;
        try {
            System.out.println("hello");
            File f = new File("D:/coding30minsAday/textFiles/csvDay76J.csv");
            if (f.exists()) {
                Scanner fScan = new Scanner(f);
                while(fScan.hasNextLine()) {
                    fScan.nextLine();
                    row++;
                }
                fScan.close();
            }
            FileWriter w = new FileWriter("D:/coding30minsAday/textFiles/csvDay76J.csv");
            w.append("\n");
            w.close();
            row++;
        } catch (Exception e) {
            System.err.println("Error");
        }
        return row;
    }
}
